.github/workflows/build.yml

```diff
@@ -397,6 +397,29 @@ jobs:
     - name: SSL tests
       run: ./python Lib/test/ssltests.py
 
+  build-android:
+    name: Android (${{ matrix.arch }})
+    needs: build-context
+    if: needs.build-context.outputs.run-tests == 'true'
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Use the same runs-on configuration as build-macos and build-ubuntu.
+          - arch: aarch64
+            runs-on: ${{ github.repository_owner == 'python' && 'ghcr.io/cirruslabs/macos-runner:sonoma' || 'macos-14' }}
```

Comment:
@mhsmith something just occurred to me: is macOS *required* for this build? GHA has Ubuntu on arm nowadays. Is this just for extended CI coverage?

Reply:
Yes, it's to check that it can be built on both Linux and Mac. It's a cross-compiler, so the architecture doesn't need to match for building, only for testing. Unfortunately we can't test aarch64 yet, because GitHub Actions doesn't support the Android emulator on Mac aarch64, and Google doesn't provide the Android development tools at all for Linux aarch64. So Android aarch64 is only tested by the buildbot.